Conceptpower+Spring/src/main/java/edu/asu/conceptpower/users/impl/UsersManager.java

```diff
@@ -30,10 +34,21 @@
 	@Autowired
 	private UserDatabaseClient client;
 	private Map<String, String> admins;
-
+	
+	
+	
 	@PostConstruct
-	public void init() {
+	public void init() throws IOException {
 		admins = new HashMap<String, String>();
+		
+		Properties prop = new Properties();
+		InputStream is = this.getClass().getResourceAsStream("/user.properties");
```

Comment:
use spring to read the properties file; you can annotate classes to read specific files (qstore might have an example, or quadriga too I believe).